tools/restatectl/src/commands/log/reconfigure.rs

```diff
@@ -26,13 +27,14 @@ use restate_types::replication::{NodeSet, ReplicationProperty};
 use restate_types::{GenerationalNodeId, PlainNodeId};
 
 use crate::connection::ConnectionInfo;
+use crate::util::RangeParam;
 
 #[derive(Run, Parser, Collect, Clone, Debug)]
 #[cling(run = "reconfigure")]
 pub struct ReconfigureOpts {
     /// LogId/Partition to seal and extend
-    #[clap(long, short)]
-    log_id: u32,
+    #[clap(long, short, required = true)]
+    log_id: Vec<RangeParam>,
```

Comment:
Can we do the same for the `logs trim` command?

Reply:
Done!